packages/sources/tradermade/src/endpoint/batch.ts

```diff
@@ -0,0 +1,109 @@
+import { Validator, Requester, util } from '@chainlink/ea-bootstrap'
+import {
+  ExecuteWithConfig,
+  Config,
+  AdapterRequest,
+  AxiosResponse,
+  InputParameters,
+} from '@chainlink/types'
+import { NAME } from '../config'
+
+/**
+ * This endpoint is similar to live but is supposed to only be used to fetch forex data.  This is why quote is a required parameter.
+ * The reason for this split is that we want to enable WS for this endpoint but not for live.
+ */
+
+export const supportedEndpoints = []
+// NOTE: unused pending tradermade service agreement
+export const batchablePropertyPath = [{ name: 'base' }, { name: 'quote' }]
+
+export const inputParameters: InputParameters = {
+  base: {
+    aliases: ['from', 'symbol', 'market'],
+    required: true,
+    description: 'The symbol of the currency to query',
+  },
+  quote: {
+    aliases: ['to', 'market', 'convert'],
+    required: true,
```

Comment:
Can't have same alias "market" on both input params

Reply:
Woah, nice catch.

Reply:
Must've missed it the first time though...

Reply:
Yeah that has been in there for ~4 months :sweat_smile:

Reply:
https://app.shortcut.com/chainlinklabs/story/26519/unit-test-for-duplicate-input-parameter-aliases